templates/get_started.md

```diff
@@ -1,16 +1,24 @@
 # Get started with Lean
```

Comment:
I think (some) research mathematicians won't understand the difference between Lean and Mathlib. Maybe this can say: "Get started with Lean and Mathlib" (so they know it's a two-step process?)